### PR TITLE
Set pack version to 3

### PR DESCRIPTION
**Pull request by gudenau**

### CODE REVIEW

File: pack.mcmeta [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "pack": {
-    "pack_format": 2,
+    "pack_format": 3,
```
Comment (InsomniaKitten) [inline pack.mcmeta]: This change will break `soulshardstow/blockstates/BlockCage` and `enderio/blockstates/blockIngotStorage`

Comment (ChloeDawn) [inline pack.mcmeta:3]: Pack format 3 enforces lower-casing, and CTP has non-lower-case files. This change will break `soulshardstow/blockstates/BlockCage` and `enderio/blockstates/blockIngotStorage`

**Review by ChloeDawn — CHANGES_REQUESTED** [review]

**Review by Drullkus — CHANGES_REQUESTED** [review]
Unfortunately I’d like to keep this version of the pack on the repo compatible with 1.10, and then pack to 1.12.2 editions locally for Curseforge.

**Comment by gudenau** [comment]
Alrighty.

**Pull request closed without merging**